poolapi/apijson.go

```diff
@@ -0,0 +1,41 @@
+package poolapi
+
+type Response struct {
+	Status  string      `json:"status"`
+	Message string      `json:"message"`
+	Data    interface{} `json:"data,omitempty"`
```

Comment:
Actually, I'm not sure how this would work though.  See `NewResponse`.  The two types presently passed to `data` are the two other new structs `PurchaseInfo` and `Stats`.  (Not maps).